style.css

```diff
@@ -0,0 +1,156 @@
+
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+body {
+  font-family: 'Museo Sans Cyrl 500';
+  font-style: normal;
+  font-size: 16px;
+  line-height: 19px;
+  color: #F8EEE4;
```

Comment:
Codacy found an issue: [Expected "#F8EEE4" to be "#f8eee4" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/MyBali/pullRequest?prid=5781003)